envi/archs/arm/disasm.py

```diff
@@ -1837,7 +1843,7 @@ def getOperAddr(self, op, emu=None):
         # if U==0, subtract
         addval *= pom
 
-        addr = (Rn + addval) & e_bits.u_maxes[self.tsize]
+        addr = Rn + addval
```

Comment:
Yes was unintentional. Fixed on my copy will push today

Date: Fri, 3 Jun 2016 05:54:26 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Update unit tests for opcodes (#3)

In envi/archs/arm/disasm.py:

> @@ -1837,7 +1843,7 @@ def getOperAddr(self, op, emu=None):
>          # if U==0, subtract
>          addval *= pom
>  
> -        addr = (Rn + addval) & e_bits.u_maxes[self.tsize]
> +        addr = Rn + addval

please put this back as you found it.  i believe you accidentally made this change.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.